lib/bin/gpa-games-cli

```diff
@@ -67,8 +67,7 @@ class HexCliGameState(UiGameState):
             raise("Unrecognized player, \"{}\"".format(ui_player))
 
     def reset_state(self):
-        self.state.reset(*self.state.board.size())
-        return self
+        self.state.reset()
```

Comment:
I didn't know before but classes that implement this type of chaining are called *fluent interfaces*, just FYI.

Thanks @JesseH-, I hadn't thought about the ambiguity between returning `self` and returning a copy being an issue. This isn't so much of a problem in other languages like Ruby and Scheme where mutation methods usually have a `!` in their names. And in Smalltalk, the default return value is actually `self`.

This stack overflow post, <http://stackoverflow.com/questions/36250990/return-self-in-python>, discusses this more. Apparently, Guido van Rossum has addressed this directly and he talks about `list.sort` as well.

While I still like the idea of fluent interfaces, I can accept that it's better to write Python without them.